src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -10,30 +10,30 @@
 {%- endblock form_widget_simple %}
 
 {% block button_widget -%}
-    {% set attr = attr|merge({class: (attr.class|default('btn-default') ~ ' btn')|trim}) %}
+    {%- set attr = attr|merge({class: (attr.class|default('btn-default') ~ ' btn')|trim}) -%}
     {{- parent() -}}
 {%- endblock button_widget %}
 
 {% block checkbox_widget -%}
     {%- set parent_label_class = parent_label_class|default(label_attr.class|default('')) -%}
-    {% if 'checkbox-inline' in parent_label_class %}
+    {%- if 'checkbox-inline' in parent_label_class -%}
```

Comment:
Are you sure those added `-` are needed? Does they make a difference as there are already `-` signs at the end of the line before and at the beginning of the next line?

Reply:
actually they're not needed, but i tried to keep it similar to the bs4 layout, will remove them in both templates.